bc/test/testVotes.js

```diff
@@ -20,25 +20,37 @@ contract('Upvotes', async accounts => {
         assert.equal(upvoteCountAfter, 1);
         upvoteCountForPost = await upvotes.getUpvoteCountForPost(postHash);
         assert.equal(upvoteCountForPost, 1);
-    })
+    });
 
     it('Should not mix with downvotes', async () => {
-        let upvotes = await Upvotes.deployed();
-        let downvotes = await Downvotes.deployed();
-        let upvoteCountBefore = await upvotes.getUpvoteCount();
+        const upvotes = await Upvotes.deployed();
+        const downvotes = await Downvotes.deployed();
+        const upvoteCountBefore = await upvotes.getUpvoteCount();
         assert.equal(upvoteCountBefore, 1);
+        const postHash = web3.utils.keccak256('some post');
+        let upvoteCountForPost = await upvotes.getUpvoteCountForPost(postHash);
+        let downvoteCountForPost = await downvotes.getDownvoteCountForPost(postHash);
+        assert.equal(upvoteCountForPost, 1);
+        assert.equal(downvoteCountForPost, 0);
+
+        const upvoteHash = web3.utils.keccak256('upvote');
+        const downvoteHash = web3.utils.keccak256('downvote');
 
-        let postHash = web3.utils.keccak256('some post');
-        await upvotes.vote(postHash, {
+        await upvotes.vote(upvoteHash, postHash, {
             value: web3.utils.toWei('0.001', 'ether')
         });
-        await downvotes.vote(postHash, {
+        await downvotes.vote(downvoteHash, postHash, {
             value: web3.utils.toWei('0.001', 'ether')
         });
 
         let upvoteCountAfter = await upvotes.getUpvoteCountForPost(postHash);
         assert.equal(upvoteCountAfter, 2);
         let downvoteCountAfter = await downvotes.getDownvoteCountForPost(postHash);
         assert.equal(downvoteCountAfter, 1);
```

Comment:
change let to const and I believe you don't need the reset of this test, am I right?

Reply:
Certainly, I'm not sure why is the rest even here xD I must have deleted something in the meantime.